include/cuco/detail/hash_functions/xxhash.cuh

```diff
@@ -314,13 +314,13 @@ struct XXHash_64 {
     // remaining data can be processed in 4-byte chunks
     if constexpr (((nbytes % 32) % 8) >= 4) {
       for (; offset <= nbytes - 4; offset += 4) {
-        h64 ^= (blocks4[offset / 4] & 0xFFFFFFFFULL) * prime1;
+        h64 ^= (blocks4[offset / 4] & 0xFFFFFFFFull) * prime1;
```

Comment:
```suggestion
        h64 ^= (blocks4[offset / 4] & 0xffffffffull) * prime1;
```

Reply:
fixed in [00e5b54](https://github.com/NVIDIA/cuCollections/pull/313/commits/00e5b54eb31aba90d396552c2552c3492c98fe0f)